qcodes_contrib_drivers/drivers/BlueFors/BlueFors.py

```diff
@@ -0,0 +1,200 @@
+                               docstring  = 'Temperature of the 50K plate',
+                               )
+
+        if channel_4k_plate is not None:
+            self.add_parameter(name       = 'temperature_4k_plate',
+                               unit       = 'K',
+                               get_parser = float,
+                               get_cmd    = lambda: self.get_temperature(2),
+                               docstring  = 'Temperature of the 4K plate',
+                               )
+
+        if channel_magnet is not None:
+            self.add_parameter(name       = 'temperature_magnet',
+                               unit       = 'K',
+                               get_parser = float,
+                               get_cmd    = lambda: self.get_temperature(3),
+                               docstring  = 'Temperature of the magnet',
+                               )
+
+        if channel_still is not None:
+            self.add_parameter(name       = 'temperature_still',
+                               unit       = 'K',
+                               get_parser = float,
+                               get_cmd    = lambda: self.get_temperature(6),
+                               docstring  = 'Temperature of the still',
+                               )
+
+        if channel_mixing_chamber is not None:
+            self.add_parameter(name       = 'temperature_mixing_chamber',
+                               unit       = 'K',
+                               get_parser = float,
+                               get_cmd    = lambda: self.get_temperature(5),
+                               docstring  = 'Temperature of the mixing chamber',
+                            )
+        
+        self.connect_message()
+
+
+    def get_temperature(self, channel: int) -> float:
+        """
+        Return the last registered temperature of the current day for the
+        channel.
+        
+        Args:
+            channel (int): Channel from which the temperature is extracted.
+
+        Returns:
+            temperature (float): Temperature of the channel in Kelvin.
+        """
+        
+        folder_name = date.today().strftime("%y-%m-%d")
+        file_path = os.path.join(self.folder_path, folder_name, 'CH'+str(channel)+' T '+folder_name+'.log')
+        
+        df = pd.read_csv(file_path,
+                            delimiter = ',',
+                            names     = ['date', 'time', 'y'],
+                            header    = None)
+
+        # There is a space before the day
+        df.index = pd.to_datetime(df['date']+'-'+df['time'], format=' %d-%m-%y-%H:%M:%S')
+        
+        return df.iloc[-1]['y']
+
+
+    def get_pressure(self, channel: int) -> float:
+        """
+        Return the last registered pressure of the current day for the
+        channel.
+        
+        Args:
+            channel (int): Channel from which the pressure is extracted.
```

Comment:
there's no need to specify the type in the docstring if it's in the function signature
```suggestion
            channel: Channel from which the pressure is extracted.
```

Reply:
My bad, I am using the "python docstring generator" on vscode and the type was added automatically...